org.fcitx.Fcitx5.Addon.McBopomofo.metainfo.xml.in

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="addon">
+  <id>org.fcitx.Fcitx5.Addon.McBopomofo</id>
+  <extends>org.fcitx.Fcitx5</extends>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <name>McBopomofo for Fcitx 5</name>
+  <summary>A phonetic input method that perdict words intelligently</summary>
```

Comment:
One more nit: s/perdict/predicts/ (spelling and grammar). I would still call it "A Bopomofo input method" as Bopomofo is a well-known term. So how about: `A Bopomofo input method that predicts phrases intelligently`?